### PR TITLE
Actually populate the view host proxy

### PR DESCRIPTION
**Pull request by abarth**
Now that the engine lets us take the view host handle, we can actually
access the view host from Dart.

### CODE REVIEW

**Comment by abarth** [comment]
@j9brown @chinmaygarde 

In theory, this should be the last step needed for basic embedding of Mozart views.  Now, I haven't actually tried it, so we'll see what really happens :)

**Comment by abarth** [comment]
This patch depends on https://github.com/flutter/engine/pull/2348

File: packages/flutter/lib/src/rendering/child_view.dart [hunk 1]
```diff
@@ -10,13 +11,28 @@ import 'package:mojo_services/mojo/ui/layouts.mojom.dart' as mojom;
...
 
 import 'box.dart';
 import 'object.dart';
 
-// TODO(abarth): Populate the view host.
-final mojom.ViewHost _viewHost = null;
+mojom.ViewHostProxy _initViewHostProxy() {
+  try {
+    return new mojom.ViewHostProxy.fromHandle(
+      new core.MojoHandle(internals.takeViewHostHandle()));
```
Comment (Hixie) [inline packages/flutter/lib/src/rendering/child_view.dart]: newline before the last )

File: packages/flutter/lib/src/rendering/child_view.dart [hunk 2]
```diff
@@ -10,13 +11,28 @@ import 'package:mojo_services/mojo/ui/layouts.mojom.dart' as mojom;
...
+  } catch (e) {
+    return null;
```
Comment (Hixie) [inline packages/flutter/lib/src/rendering/child_view.dart]: do we really want to silently catch all exceptions here? Or just one specific one?

Comment (abarth) [inline packages/flutter/lib/src/rendering/child_view.dart]: I'm not sure what the right error handling is here.  The reality is that the callers aren't actually prepared to get a null handle out, so something terrible is going to happen pretty soon.  We might want to skip the try/catch entirely.

Comment (Hixie) [inline packages/flutter/lib/src/rendering/child_view.dart]: fine by me.

**Comment by Hixie** [comment]
lgtm either way

**Pull request merged**